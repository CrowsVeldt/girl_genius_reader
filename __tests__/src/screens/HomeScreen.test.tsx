import React from "react";
import renderer from "react-test-renderer";

import Home from "../../../src/screens/HomeScreen";
import ComicProvider from "../../../src/context/ComicContext";

describe("<Home />", () => {
  //it("renders without change", () => {
  //const tree = renderer
  //.create(
  //<ComicProvider>
  //<Home />
  //</ComicProvider>
  //)
  //.toJSON();
  //expect(tree).toMatchSnapshot();
  //});
});
