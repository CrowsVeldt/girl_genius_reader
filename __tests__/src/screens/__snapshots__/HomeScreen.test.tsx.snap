// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> renders without change 1`] = `
<View
  style={
    {
      "alignContent": "center",
      "backgroundColor": undefined,
      "height": 1334,
      "paddingTop": 50,
      "width": 750,
    }
  }
>
  <RCTScrollView
    contentContainerStyle={
      {
        "height": 1184,
        "width": 750,
      }
    }
  >
    <View />
  </RCTScrollView>
</View>
`;
